packages/framework/src/Model/Statistics/StatisticsRepository.php

```diff
@@ -67,14 +69,15 @@ public function getNewOrdersCountByDayBetweenTwoDateTimes(DateTime $start, DateT
         $query = $this->em->createNativeQuery(
             'SELECT DATE(o.created_at) AS date, COUNT(o.created_at) AS count
             FROM orders o
-            WHERE o.created_at BETWEEN :start_date AND :end_date
+            WHERE o.created_at BETWEEN :start_date AND :end_date AND o.status_id != :canceled
```

Comment:
if this is some fix, can you please put it into separate commit?

Reply:
removed